cms/utils/page.py

```diff
@@ -191,13 +191,15 @@ def get_all_pages_from_path(site, path, language):
     return pages.filter(title_set__language=language)
 
 
-def get_available_slug(site, path, language, suffix='copy', modified=False):
+def get_available_slug(site, path, language, suffix='copy', modified=False, current=None):
```

Comment:
I think my only comment on this change would be that `current` is ambiguous. I know it's part of the page utils, but `current_page` would perhaps make things more obvious.